common/data_refinery_common/message_queue.py

```diff
@@ -18,13 +18,12 @@
 NONE_JOB_ERROR_TEMPLATE = "send_job was called with NONE job_type: {} for {} job {}"
 
 
-def send_job(job_type: Enum, job_id: int) -> None:
+def send_job(job_type: Enum, job) -> None:
```

Comment:
I think there are a few places elsewhere that call this function that have not had their signatures updated - transcriptome_index, API smash dispatcher,